data.js

```diff
@@ -739,4 +739,48 @@ exports.data = [
             updatedBy: "Alexander", updatedAt: "2019-12-21"
         },
     },
+    {
+        meta: {
+            name: "Climate Reality Project",
+            link: "https://www.climaterealityproject.org/",
+            label: "climate-realtiy-project",
+        },
+        description: {
+            content: "Founded by former Vice President Al Gore Climate Reality Project tries to reach their vision of a global network of climate activists.",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },
+        established: {
+            content: "(2006)[https://www.climaterealityproject.org/whoweare]",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },   
+        kpis: {
+                content: "- presentations / pa \n- new climate reality leaders / pa \n- downloaded resources / pa",
```

Comment:
Suggest `- presentations\n- newly trained climate reality leaders (people able to hold talks and spread the word)\n`

Reply:
changed to wording